Inko 136 client statement

mkalema opened the pull request:
## Description

Describe the changes made and why they were made.

Ignore if these details are present on the associated [Apache Fineract JIRA ticket](https://github.com/apache/fineract/pull/1284).


## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Write the commit message as per https://github.com/apache/fineract/#pull-requests

- [ ] Acknowledge that we will not review PRs that are not passing the build _("green")_ - it is your responsibility to get a proposed PR to pass the build, not primarily the project's maintainers.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Follow coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions.

- [ ] Add required Swagger annotation and update API documentation at fineract-provider/src/main/resources/static/legacy-docs/apiLive.htm with details of any API changes

- [ ] Submission is not a "code dump".  (Large changes can be made "in repository" via a branch.  Ask on the developer mailing list for guidance, if required.)

FYI our guidelines for code reviews are at https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide.

mkalema committed:
WIP: INKO-136 added client statement report query

mkalema committed:
INKO-136 added client statement report

hero78 reviewed (changes requested):
@mkalema  For Loan account status, We added 9 More stages. And they are activated by the global configuration.
Let's have a call I take you through this

mkalema commented:
> @mkalema For Loan account status, We added 9 More stages. And they are activated by the global configuration. Let's have a call I take you through this

I need to find these and udate report query. Let's link up

mkalema committed:
INKO-136 updated report sql query

mkalema committed:
INKO-136 updated client statement report

mkalema commented:
> @mkalema For Loan account status, We added 9 More stages. And they are activated by the global configuration. Let's have a call I take you through this

@hero78 i have updated the report and incorporated loan decision in Status
